hw/ip_templates/rv_plic/doc/programmers_guide.md

```diff
@@ -6,7 +6,7 @@ After reset, RV_PLIC doesn't generate any interrupts to any targets even if
 interrupt sources are set, as all priorities and thresholds are 0 by default and
 all ``IE`` values are 0. Software should configure the above three registers.
 
-[`PRIO0`](../data/rv_plic.hjson#prio0) .. [`PRIO31`](../data/rv_plic.hjson#prio1) registers are unique. So, only one of the targets
+[`PRIO0`](../data/rv_plic.hjson#prio_0) .. [`PRIO31`](../data/rv_plic.hjson#prio_1) registers are unique. So, only one of the targets
```

Comment:
Probably a typo that existed before, but shouldn't it be
```suggestion
[`PRIO0`](../data/rv_plic.hjson#prio_0) .. [`PRIO31`](../data/rv_plic.hjson#prio_31) registers are unique. So, only one of the targets
```
?

Reply:
This type of link appears to be broken in general, so I think I'll skip fixing this in this particular PR. alert_handler has a similar setup trying to reference data from the hjson, but it doesn't get transformed into a link that goes to the correct HTML page.

In addition, the programming guide should probably be templated to use the correct range of CSRs.